docroot/modules/custom/uiowa_core/uiowa_core.module

```diff
@@ -61,6 +61,40 @@ function uiowa_core_form_alter(&$form, FormStateInterface $form_state, $form_id)
       }
       break;
 
+    // @todo Move this to a more appropriate location.
+    case 'node_area_of_study_form':
+      foreach ([
+        'field_area_of_study_majors',
+        'field_area_of_study_minors',
+        'field_area_of_study_certificates',
+        'field_area_of_study_preprof',
+        'field_area_of_study_online',
+      ] as $program_type) {
+
+        if (isset($form[$program_type])) {
+          $program_toggle = $program_type . '_toggle';
+          $form[$program_toggle] = [
+            '#type' => 'checkbox',
+            '#title' => $form[$program_type]['widget']['#title'],
+            '#weight' => $form[$program_type]['#weight'] - 1,
+            '#name' => $program_toggle,
+          ];
+
+          $form['#group_children'][$program_toggle] = 'group_area_of_study_program_type';
```

Comment:
I'm getting a WSOD error when trying to add new areas of study content from this line. Commenting it out, things seem to work as expected, but not sure if leaving it out has other effects I'm not noticing.

`The website encountered an unexpected error. Please try again later.
Error: Unsupported operand types in Drupal\field_group\FieldGroupFormatterPluginManager->prepareConfiguration() (line 114 of modules/contrib/field_group/src/FieldGroupFormatterPluginManager.php).`

Reply:
I think I may need to re-export config.

Reply:
I think this may be resolved by the last commit.

Reply:
That did it! 👍